### PR TITLE
Upgraded to django version 1.11.5

### PR DESCRIPTION
**Pull request by party98**
Resolves issue #2

### CODE REVIEW

File: src/vehicles/views.py [hunk 1]
```diff
@@ -97,37 +88,29 @@ def category_page(request, slug):
         vehicles = paginator.page(page)
     except (InvalidPage, EmptyPage):
         vehicles = paginator.page(paginator.num_pages)
-
+    print vehicles
```
Comment (haroon-sheikh) [inline src/vehicles/views.py]: Can you please remove the print statements?

**Comment by haroon-sheikh** [comment]
@party98 can you please have a look at the failing tests? `python manage.py test`

**Comment by party98** [comment]
@haroon-sheikh there seems to be errors in the tests itself which is causing the tests to fail

**Comment by haroon-sheikh** [comment]
@party98 do you think, you can have a go at updating these?

**Comment by party98** [comment]
@haroon-sheikh started already.
Almost done

**Pull request merged**